redaxo/src/addons/mediapool/lib/service_media.php

```diff
@@ -343,7 +343,7 @@ public static function getList(array $filter = [], array $orderBy = [], ?rex_pag
         }
 
         if ($pager) {
-            $query .= ' ORDER BY ' . implode(', ', $orderbys);
+            $query .= ' GROUP BY m.filename ORDER BY ' . implode(', ', $orderbys);
             $sql->setQuery(str_replace('SELECT m.filename', 'SELECT count(*)', $query), $queryParams);
             $pager->setRowCount((int) $sql->getValue('count(*)'));
 
```

Comment:
Ich würde es anders lösen. Das ORDER-BY ist für die Count-Query überflüssig:

```suggestion
            $sql->setQuery(str_replace('SELECT m.filename', 'SELECT count(*)', $query), $queryParams);
            $pager->setRowCount((int) $sql->getValue('count(*)'));

            $query .= '  ORDER BY ' . implode(', ', $orderbys);
```